### PR TITLE
MAUI-3664-DateNavigationUGContent_MAUI_SfCalendar

### PR DESCRIPTION
**Pull request by Priyadharshini-Periyasamy**
## *Description*

- Need to add Date navigation UG content for MAUI SfCalendar

### CODE REVIEW

File: MAUI/Calendar/date-navigations.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+description: Learn here all about Date navigations feature of Syncfusion MAUI Calendar (SfCalendar) widget and more.
+platform: maui
+control: SfCalendar
+documentation: ug
+---
+
+# Date Navigations in MAUI Calendar (SfCalendar)
+
+## Programmatic date navigation
+Allows you to navigate through the Dates by programmatically in the calendar widget by using the [DisplayDate](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html#Syncfusion_Maui_Calendar_SfCalendar_DisplayDate) property of [SfCalendar](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html).
```
Comment (laksmipriyamurugesan) [inline MAUI/Calendar/date-navigations.md:13]: Programmatic date navigation
It allows you to navigate using the Dates programmatically in the calendar widget using the [DisplayDate](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html#Syncfusion_Maui_Calendar_SfCalendar_DisplayDate) property of [SfCalendar](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html).

File: MAUI/Calendar/date-navigations.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+this.Calendar.DisplayDate = DateTime.Now.AddMonths(2).Date;
+
+{% endhighlight %}
+{% endtabs %}
+
+![display-date-in-maui-calendar](images/date-navigation/display-date-in-maui-calendar.png)
+
+## Programmatic view navigation
+Allows you to navigate through the views by programmatically in the calendar widget by using the [View](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html#Syncfusion_Maui_Calendar_SfCalendar_View) property of [SfCalendar](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html).
```
Comment (laksmipriyamurugesan) [inline MAUI/Calendar/date-navigations.md:31]: It allows you to navigate using the views programmatically in the calendar widget using the [View](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html#Syncfusion_Maui_Calendar_SfCalendar_View) property of [SfCalendar](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html).

File: MAUI/Calendar/date-navigations.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+
+this.Calendar.View = CalendarView.Month;
+
+{% endhighlight %}
+{% endtabs %}
+
+![monthview-in-maui-calendar](images/date-navigation/monthview-in-maui-calendar.png)
+
+## Allow view navigation
+Allows you to navigate by using the `AllowViewNavigation` property through tap interaction on the cell or header. By using this property you can able to restrict the view navigation and allow you to select the cells in the Year, Decade and Century views.
```
Comment (laksmipriyamurugesan) [inline MAUI/Calendar/date-navigations.md:49]: It allows you to navigate using the `AllowViewNavigation` property by tapping interaction on the cell or header. By using this property, you can restrict the view navigation and allow you to select the cells in the Year, Decade, and Century views.

File: MAUI/Calendar/date-navigations.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+
+The following code shows whem the [AllowViewNavigation](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html#Syncfusion_Maui_Calendar_SfCalendar_AllowViewNavigation) property is `true`.
```
Comment (laksmipriyamurugesan) [inline MAUI/Calendar/date-navigations.md:51]: whem-->when

File: MAUI/Calendar/date-navigations.md [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.AllowViewNavigation = true;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Programmatically change to adjacent dates
+The next and previous view can be navigated by swiping the Calendar control from right to left and left to right. In the `SfCalendar`, view can be changed programmatically by using the [Forward](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html#Syncfusion_Maui_Calendar_SfCalendar_Forward) and [Backward](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html#Syncfusion_Maui_Calendar_SfCalendar_Backward) methods.
```
Comment (laksmipriyamurugesan) [inline MAUI/Calendar/date-navigations.md:70]: The next and previous views can be navigated by swiping the Calendar control from right to left and left to right. In the `SfCalendar,` view can be changed programmatically by using the [Forward](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html#Syncfusion_Maui_Calendar_SfCalendar_Forward) and [Backward](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html#Syncfusion_Maui_Calendar_SfCalendar_Backward) methods.

File: MAUI/Calendar/date-navigations.md [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+private void button_Clicked(object sender, EventArgs e)
+{
+  this.Calendar.Backward();
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Navigation direction
+Views can be navigated by using the `Navigation direction` property either `Vertical` or `Horizontal` directions by setting the `NavigationDirection` property of `Calendar`. The default [NavigationDirection](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html#Syncfusion_Maui_Calendar_SfCalendar_NavigationDirection) is `Vertical`.
```
Comment (laksmipriyamurugesan) [inline MAUI/Calendar/date-navigations.md:129]: Views can be navigated by using the `Navigation direction` property, either `Vertical` or `Horizontal` direction by setting the `NavigationDirection` property of `Calendar.` The default [NavigationDirection](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html#Syncfusion_Maui_Calendar_SfCalendar_NavigationDirection) is `Vertical.`

File: MAUI/Calendar/date-navigations.md [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+
+The following code shows the Navigation direction in `Horizontal`,
```
Comment (laksmipriyamurugesan) [inline MAUI/Calendar/date-navigations.md:131]: The following code shows the Navigation in the `Horizontal` direction.

File: MAUI/Calendar/date-navigations.md [hunk 8]
```diff
@@ -0,0 +1,172 @@
...
+
+this.Calendar.NavigationDirection = CalendarNavigationDirection.Horizontal;
+
+{% endhighlight %}
+{% endtabs %}
+
+![horizontal-navigation-direction-in-maui-calendar](images/date-navigation/horizontal-navigation-direction-in-maui-calendar.png)
+
+## Show navigation arrow
+By using the `ShowNavigationArrows` property of the `Calendar` you can navigate to the next or previous views of the calendar without swiping. By default, the value of [ShowNavigationArrows](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.CalendarHeaderView.html#Syncfusion_Maui_Calendar_CalendarHeaderView_ShowNavigationArrows) is `true`.
```
Comment (laksmipriyamurugesan) [inline MAUI/Calendar/date-navigations.md:152]: By using the `ShowNavigationArrows` property of the `Calendar,` you can navigate to the next or previous view of the calendar without swiping. By default, the value of [ShowNavigationArrows](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.CalendarHeaderView.html#Syncfusion_Maui_Calendar_CalendarHeaderView_ShowNavigationArrows) is `true.`

File: MAUI/Calendar/date-navigations.md (hunk 1 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/date-navigations.md]: Content changed

File: MAUI/Calendar/date-navigations.md (hunk 2 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/date-navigations.md]: Content changed

File: MAUI/Calendar/date-navigations.md (hunk 3 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/date-navigations.md]: Content modified.

File: MAUI/Calendar/date-navigations.md (hunk 4 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/date-navigations.md]: Whem is changed to when

File: MAUI/Calendar/date-navigations.md (hunk 5 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/date-navigations.md]: Content changed

File: MAUI/Calendar/date-navigations.md (hunk 6 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/date-navigations.md]: Content changed

File: MAUI/Calendar/date-navigations.md (hunk 7 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/date-navigations.md]: Content modified

File: MAUI/Calendar/date-navigations.md (hunk 8 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/date-navigations.md]: Content changed

File: MAUI/Calendar/date-navigations.md [hunk 9]
```diff
@@ -0,0 +1,172 @@
+---
+layout: post
+title: Date Navigations in  MAUI Calendar control | Syncfusion
+description: Learn here all about Date navigations feature of Syncfusion .NET MAUI Calendar (SfCalendar) control and more.
+platform: maui
+control: SfCalendar
+documentation: ug
+---
+
+# Date Navigations in MAUI Calendar (SfCalendar)
```
[the remark this answers is not in the document]
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/date-navigations.md]: Changed to .NET MAUI Calendar

**Review by Sasikumar3595 — APPROVED** [review]
This is fine.

**Review by Nijamudeen-Mohamed — APPROVED** [review]
Fine

**Pull request merged**